zip-0226.rst

```diff
@@ -324,10 +345,17 @@ Transaction Fees
 
 The fee mechanism for the upgrades proposed in this ZIP will follow the mechanism described in ZIP 317 for the ZSA protocol upgrade [#zip-0317b]_.
 
-Deployment
-==========
+Backward Compatibility
+----------------------
+
+In order to have backward compatibility with the ZEC notes, we have designed the circuit to support both ZEC and ZSA notes. As we specify above, there are three main reasons we can do this:
+- The input notes with an Asset Base denote the Custom Assets, generating a note commitment that includes the Asset Identifier; whereas the notes without an Asset Base denote the ZEC notes from before the protocol upgrade, and generate a note commitment that does not include the Asset Base, in order to maintain the referencability to the Merkle tree
+- The value commitment is abstracted to allow for the value base-point as a variable private input to the proof
+- The ZEC-based Actions will still include dummy input notes, whereas the ZSA-based Actions will include split input notes.
```

Comment:
The ZEC-based Actions will still include dummy input notes, whereas the ZSA-based Actions will include both dummy and split input notes.

Reply:
Done.